src/contact.js

```diff
@@ -14,6 +14,7 @@ class Contact {
     this.id = o.id;
     this.mdid = o.mdid;
     this.name = o.name;
+    this.senderName = o.senderName;
```

Comment:
Ok I will just do what Android does. I will also remove initiator_source since mobile teams also decided to not use that.